core/templates/dev/head/player/exploration_player.html

```diff
@@ -105,6 +107,92 @@
+          <li>
+            <span class="fa fa-eye fa-lg"></span><[viewsCount]></li>
+          <li>
+            <span class="fa fa-clock-o fa-lg"></span><[lastModifiedDate | date: 'MM/dd/yyyy']></li>
+          <ul layout="row" layout-align="space-between center" class="oppia-exploration-contributors-profile">
+            <img src="/third_party/static/material-design-icons-1.0.1/ic_group_black_48dp.png">
+            <li ng-repeat="contributor in explorationContributors| limitTo: 2">
+              <img class="img-circle" ng-src="/images/general/user_mint_48px.png" alt=""></li>
+
+            <!--TODO(Makonda) get profile picture and profile url to make image click direct to contributor profile-->
+            
+            <li ng-if='explorationContributors.length > 2' class="oppia-contributors-more-circle dropdown">
+              <a href="" class="dropdown-toggle"
+                ng-mouseover="onMouseoverDropdownMenu($event)" data-toggle="dropdown">
+                <[explorationContributors.length - 2]>+</a>
+              <ul class="dropdown-menu oppia-contributors-dropdown" role="menu"
+                ng-mouseleave="onMouseoutDropdownMenu($event)">
+                <li ng-repeat="conributor in explorationContributors | limitTo: explorationContributors.length-2: 3">
+                  <img class="img-circle" ng-src="/images/general/user_blue_72px.png" alt="">
+                </li>
+              </ul>
+             </li>
+          </ul>
+        </ul>
+        <div class="oppia-social-media-and-tags" layout="row">
+          <div flex="55">
+            <md-content layout="row">
+              <div class="oppia-tag-icon"><span class="fa fa-tags fa-lg"></span></div>
+              <md-chips layout-align="center center">
+                  <md-chip ng-if='explorationTags.length > 2' class="oppia-information-card-tooltip-more"
+                   tooltip-append-to-body=true tooltip="<[explorationTags.slice(2).join()]>"
+                   tooltip-placement="right">
+                    <[explorationTags.length - 2]>+ more tags</md-chip>
+                  <md-chip><[explorationTags.slice(0,2).join()]></md-chip>
+                  <md-chip ng-if="explorationTags.length < 1">This exploration has no tags<md-chip>
+              </md-chips>
+            </md-content>
+          </div>
+          <div flex="45">
+            <ul layout="row" layout-align="end center">
+              <li>
+                <a href="" ng-click="showEmbedExplorationModal(explorationId)">
+                  <img src="/third_party/static/material-design-icons-1.0.1/ic_share_black_48dp.png">
+                </a>
+              </li>
+              <li>
+                <a ng-href="https://plus.google.com/share?url=<[serverName]>/explore/<[explorationId]>" onclick="return !window.open(this.href, '', 'height=600, width=600, menubar=no, toolbar=no, resizable=yes, scrollbars=yes')" target="_window">
```

Comment:
Surround each of these by the appropriate `if SHARING_OPTIONS[...]` blocks, as is done in `exploration_player.html`. We can also remove the sharing button and the existing code from the navbar, since it duplicates this.